staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/helpers.go

```diff
@@ -84,13 +81,6 @@ func (s ByVersionPriority) Less(i, j int) bool {
 	return version.CompareKubeAwareVersionStrings(s[i].Spec.Version, s[j].Spec.Version) > 0
 }
 
-// APIServiceNameToGroupVersion returns the GroupVersion for a given apiServiceName.  The name
-// must be valid, but any object you get back from an informer will be valid.
-func APIServiceNameToGroupVersion(apiServiceName string) schema.GroupVersion {
-	tokens := strings.SplitN(apiServiceName, ".", 2)
-	return schema.GroupVersion{Group: tokens[1], Version: tokens[0]}
-}
-
```

Comment:
so you moved the above function to v1.helpers, what about other functions in this file?

Reply:
actually, am not moving it. a copy is made in the v1 helpers and this internal helper got unreferenced so removed (my IDE tooling told me).